filewatcher/filewatcher.go

```diff
@@ -161,14 +227,21 @@ func (r *Result) watcherLoop(
 				continue
 			}
 
-			if filepath.Base(ev.Name) != file {
-				continue
-			}
-
 			switch ev.Op {
 			default:
 				// Ignore uninterested events.
 			case fsnotify.Create, fsnotify.Write:
```

Comment:
with the add of directory support, I think we also need to add `fsnotify.Remove` to this `case`.